server/api.py

```diff
@@ -88,9 +82,12 @@ def detail_search():
     else:
         params = [{'last_name':name},{'first_name':name}]
 
+    print params
     ids = set()
     final = []
     for param in params:
+      param['affiliation'] = 'FAC'
+    for param in params:
```

Comment:
Is this a mistake? I think you duplicated this line and indented line 89 too far.